internal/tiltfile/updatesettings/update_settings.go

```diff
@@ -27,19 +28,28 @@ func (e Extension) OnStart(env *starkit.Environment) error {
 }
 
 func (e Extension) updateSettings(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
-	var maxParallelUpdates int
+	maxParallelUpdates := model.DefaultMaxParallelUpdates
+	k8sUpsertTimeoutSecs := int(model.DefaultK8sUpsertTimeout / time.Second)
+
 	if err := starkit.UnpackArgs(thread, fn.Name(), args, kwargs,
-		"max_parallel_updates", &maxParallelUpdates); err != nil {
+		"max_parallel_updates?", &maxParallelUpdates,
+		"k8s_upsert_timeout_secs?", &k8sUpsertTimeoutSecs); err != nil {
```

Comment:
i wonder if we're eventually going to end up with a time parser... i feel like there are a few of these where the units are odd and it feels very un-kubernetes-like (where you usually specify durations as '100ms', '10s', etc)